File: Pertemuan 3/luas permukaan kubus.pas
program Menghitung_Luas_Permukaan_Kubus;

uses crt;

var
        s1,s2,luas:real;


begin
        clrscr;
        writeln('========= Menghitung Luas Permukaan Kubus =========');
        write('Masukkan nilai s1 kubus    =   ');readln(s1);
        write('Masukkan nilai s2 kubus    =   ');readln(s2);

        luas:=6*s1*s2;

        writeln('==========================================');
        writeln('Luas Permukaan Kubus dalam bentuk desimal : ',luas:4:2);
        writeln('Luas Permukaan Kubus dalam bentuk bilangan bulat : ',luas:4:0);
        readln;

end.

File: Pertemuan 3/luas permukaan kubus.cpp
#include<stdio.h>
#include<conio.h>


main(){
    float s1,s2,luas;

    printf("Masukan Nilai Sisi 1 Kubus : ");scanf("%f",&s1);
    printf("Masukan Nilai Sisi 2 Kubus : ");scanf("%f",&s2);

    luas=6*s1*s2;

    printf("Luas Permukaan Kubus Dalam Bentuk Desimal : %10.0f\n",luas);
    printf("Luas Permukaan Kubus Dalam Bentuk Bilangan Bulat : %10.3f\n",luas);
    getch();
    
}
